### ISSUE TITLE
Security: Information leakage via REST-API

### ISSUE THREAD

**Issue by elbird** [gentics/mesh-incubator#165]
## Gentics Mesh Version, operating system, or hardware.
- v0.9.22

## Problem
The root rest endpoint causes information leakage:  
the version information (including build id) are returned for mesh, orient-db, elastic-search and vert.x. An attacker could use this information to find security issues in these libraries for an attack on mesh.

## Reproducer
Mesh Rest-API: GET / 

## Expected behaviour 
Per default as little information as possible should be exposed to the outside. (If really needed the version information should only viewable by admins)

**Comment by bernhardriegler** [comment]
This should be configurable to avoid a breaking change.

This needs to be addressed in GraphQl as in REST

https://github.com/gentics/mesh/blob/bdb856ed845abfba60da539a451a4c3b060c78f3/core/src/main/java/com/gentics/mesh/core/endpoint/admin/RestInfoEndpoint.java#L78
IS:
```
adminHandler.handleVersions(ac);
```
SHOULD
```
adminHandler.blockingHandler(rc -> handlerUtilities.requiresAdminRole(rc));
```

**Comment by Jotschi** [comment]
I suggest to create a setting in the config that controls this behavior. Adding a perm handler would break existing implementations and I not needed here.

**Comment by Jotschi** [comment]
Server tokens can now be configured. Admin perm is required to view those.

**Comment by Jotschi** [comment]
Fixed in 1.4.7
